x-pack/plugins/cloud/public/user_menu_links.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { UserMenuLink } from '../../security/public';
+import { CloudConfigType } from '.';
+
+export const createUserMenuLinks = (config: CloudConfigType): UserMenuLink[] => {
+  const { resetPasswordUrl, accountUrl, securityUrl } = config;
+  const userMenuLinks = [] as UserMenuLink[];
+
+  if (resetPasswordUrl) {
+    userMenuLinks.push({
+      label: i18n.translate('xpack.cloud.userMenuLinks.profileLinkText', {
+        defaultMessage: 'Cloud profile',
+      }),
+      iconType: 'logoCloud',
+      href: resetPasswordUrl,
+      order: 100,
+    });
+  }
+
+  if (accountUrl) {
+    userMenuLinks.push({
+      label: i18n.translate('xpack.cloud.userMenuLinks.accountLinkText', {
+        defaultMessage: 'Account',
+      }),
+      iconType: 'gear',
+      href: accountUrl,
+      order: 200,
+    });
+  }
+
+  if (securityUrl) {
+    userMenuLinks.push({
+      label: i18n.translate('xpack.cloud.userMenuLinks.securityLinkText', {
+        defaultMessage: 'Security',
+      }),
+      iconType: 'lock',
+      href: securityUrl,
+      order: 300,
+    });
+  }
```

Comment:
I'm know there have been a lot of discussions around the link text, but I still find `Account` and `Security` quite confusing. Looking at the screenshot, these links don't give me any indication that I'd be leaving my Kibana installation to go to the cloud console.

What are your thoughts around providing all of the cloud links within a nested context menu panel, to make them more distinct?

My other thought was to change the icons to indicate that they're cloud-related, but having three `logoCloud` options next to each other might not be very straightforward either.

Reply:
There's certainly more to be sorted out here. The general premise we're after is that the top header is truly global and that we don't differentiate between Cloud UI and Kibana UI. Now, there's a lot of related work that needs to happen for this to become reality, but I would refrain from separating things out. Given that, we may want to replace the Cloud logo, ultimately.

My preference would be to get this PR merged since Catherine is headed back to Canvas 😢 , and then move our discussion to #82755 . We've got time to sort this out before the Cloud side catches up at which point this 'turns on'.